src/plugins/visualizations/public/wizard/new_vis_modal.tsx

```diff
@@ -185,4 +199,5 @@ class NewVisModal extends React.Component<TypeSelectionProps, TypeSelectionState
   }
 }
 
-export { NewVisModal };
+// eslint-disable-next-line import/no-default-export
+export { NewVisModal as default };
```

Comment:
Isn't there a way to load this asynchronously with a non-default export?

Reply:
No, it should be exported as default